Plugins/GitUIPluginInterfaces/IGitModule.cs

```diff
@@ -70,6 +74,21 @@ public interface IGitModule
         /// <summary>Indicates HEAD is not pointing to a branch (i.e. it is detached).</summary>
         bool IsDetachedHead();
 
+        /// <summary>
+        /// Convert the path for the Git executable.
+        /// For WSL Git, the path will be adjusted.
```

Comment:
```suggestion
        ///  Returns the path to the Git executable. For WSL Git the path is adjusted.
```

Reply:
Rewrote the documentation, slightly different